src/libs/actions/Session/index.ts

```diff
@@ -296,11 +296,11 @@ function beginSignIn(email: string) {
  * Given an idToken from Sign in with Apple, checks the API to see if an account
  * exists for that email address and signs the user in if so.
  */
-function beginAppleSignIn(idToken: string) {
+function beginAppleSignIn(idToken: string | undefined | null) {
```

Comment:
should this function accept undefined strings? If so, why?

```suggestion
function beginAppleSignIn(idToken: string) {
```

Reply:
@rushatgabhane The beginAppleSignIn function is called in `Promise.then` where the received value can be undefined . We should use this approach or call it in a condition